api-module-library/hubspot/tests/api.test.js

```diff
@@ -1,26 +1,376 @@
-const Manager = require('./manager');
-const mongoose = require('mongoose');
-const config = require('./defaultConfig.json');
+const {Api} = require('../api');
+const config = require('../defaultConfig.json');
+const Authenticator = require('@friggframework/test-environment/Authenticator');
 
-describe(`Should fully test the ${config.label} Manager`, () => {
-    let manager, userManager;
+describe(`${config.label} API tests`, () => {
+
+
+    const apiParams = {
+        client_id: process.env.HUBSPOT_CLIENT_ID,
+        client_secret: process.env.HUBSPOT_CLIENT_SECRET,
+        redirect_uri: `${process.env.REDIRECT_URI}/hubspot`,
+        scope: process.env.HUBSPOT_SCOPE
+    };
+    const api = new Api(apiParams);
 
     beforeAll(async () => {
-        await mongoose.connect(process.env.MONGO_URI);
-        manager = await Manager.getInstance({
-            userId: new mongoose.Types.ObjectId(),
+        const url = await api.getAuthUri();
+        const response = await Authenticator.oauth2(url);
+        const baseArr = response.base.split('/');
+        response.entityType = baseArr[baseArr.length - 1];
+        delete response.base;
+
+        await api.getTokenFromCode(response.data.code);
+    });
+
+    describe('HS User Info', () => {
+        it('should return the user details', async () => {
+            let response = await api.getUserDetails();
+            expect(response).toHaveProperty('portalId');
+            expect(response).toHaveProperty('token');
+            expect(response).toHaveProperty('app_id');
+        });
+    });
+
+    describe.skip('HS Deals', () => {
+        it('should return a deal by id', async () => {
+            let deal_id = '2022088696';
+            let response = await api.getDealById(deal_id);
+            expect(response.id).toBe(deal_id);
+            // expect(response.properties.amount).to.eq('100000');
+            // expect(response.properties.dealname).to.eq('Test');
+            // expect(response.properties.dealstage).to.eq('appointmentscheduled');
```

Comment:
Minor: it is usually good practice to remove code that isn't utilized. By the way, are we sure we want to disable these conditions, or should we just update their values (or the code that made them different)?